modules/security-context-constraints-about.adoc

```diff
@@ -52,8 +52,7 @@ effectively root on the cluster and must be trusted accordingly.
 
 [IMPORTANT]
 ====
-Do not modify the default SCCs. Customizing the default SCCs can lead to issues
-when {product-title} is upgraded.
+Do not modify the default SCCs. During upgrades between some versions of {product-title}, the values of the default SCCs are reset to the default values, which discards all customizations to those SCCs.
```

Comment:
I wouldn't dismiss the `Customizing the default SCCs can lead to issues when {product-title} is upgraded.` part, I would in fact emphasize it further: `Customizing the default SCCs can lead to issues when some of the platform pods are rolled out.` and I would also keep the part about upgrades you're adding, too.

Reply:
@stlaz, will you please take another look?